config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  root 'tasks#index'
+
```

Comment:
I'm surprised you didn't use the resources command, but it wasn't a requirement!